test/EmptyApp.FunctionalTests/project.json

```diff
@@ -1,31 +1,37 @@
 ﻿{
   "webroot": "wwwroot",
-  "version": "1.0.0-*",
+  "version": "1.0.0-rc2",
   "compilationOptions": {
     "keyFile": "../../keys/35MSSharedLib1024.snk",
-    "delaySign": true
-    },
+    "delaySign": true,
+    "preserveCompilationContext": true
+  },
   "dependencies": {
-    "Microsoft.AspNet.Server.WebListener": "1.0.0-rc1-final",
-    "Microsoft.ApplicationInsights.AspNet": "1.0.0-rc1-final",
-    "FunctionalTestUtils": "1.0.0-rc1-final",
-    "xunit.runner.dnx": "2.1.0-rc1-*",
-    "xunit": "2.1.0-rc1-*",
-    "Microsoft.AspNet.Diagnostics": "1.0.0-rc1-final"
+    "Microsoft.AspNetCore.Server.WebListener": "0.1.0-rc2-*",
+    "Microsoft.AspNetCore.Diagnostics": "1.0.0-rc2-*",
+    "Microsoft.ApplicationInsights.AspNetCore": "1.0.0-rc2-*",
+    "FunctionalTestUtils": "1.0.0-rc2"
   },
-
+  "testRunner": "xunit",
   "commands": {
-    "web": "Microsoft.AspNet.Hosting --server Microsoft.AspNet.Server.WebListener --server.urls http://localhost:5000",
-    "test": "xunit.runner.dnx"
+    "web": "Microsoft.AspNetCore.Hosting --server Microsoft.AspNetCore.Server.WebListener --server.urls http://localhost:5000",
```

Comment:
WebListener is not going to ship at RTM. Recommend changing this to use Kestrel if you need a standalone host.